bin/MadGraph5_aMCatNLO/cards/production/13TeV/tH_t_topdecay_LH_narrow_M1100/tH_t_topdecay_LH_narrow_M1100_proc_card.dat

```diff
@@ -0,0 +1,41 @@
+#************************************************************
+#*                     MadGraph5_aMC@NLO                    *
+#*                                                          *
+#*                *                       *                 *
+#*                  *        * *        *                   *
+#*                    * * * * 5 * * * *                     *
+#*                  *        * *        *                   *
+#*                *                       *                 *
+#*                                                          *
+#*                                                          *
+#*         VERSION 2.2.1                 2014-09-25         *
+#*                                                          *
+#*    The MadGraph5_aMC@NLO Development Team - Find us at   *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#*                                                          *
+#*               Command File for MadGraph5_aMC@NLO         *
+#*                                                          *
+#*     run as ./bin/mg5_aMC  filename                       *
+#*                                                          *
+#************************************************************
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+import model sm
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+import model STP_UFO_freeWidth
+define Topo = t23 t23~
+define bb = b b~
+define WW = w+ w-
+define tt = t t~
+define ferm =  ve vm vt ve~ vm~ vt~ u c d s b u~ c~ d~ s~ b~ e- mu- ta- e+ mu+ ta+
+generate p p > j tt Topo  / h ,  tt > bb WW , ( Topo > h tt / z w+ w- ,  tt > bb WW )
```

Comment:
If I understand it correctly, this is doing the decay of the bsm particle, and decaying the tops to b W, but NOT decaying the W.  Any reason not to decay the W as well?  Doing it like this will leave the W decaying in pythia, losing polarization/spin correlation effects (and possibly also not handling correctly the width of the W)